src/llmcompressor/transformers/utils/helpers.py

```diff
@@ -490,3 +492,27 @@ def download_model_directory(pretrained_model_name_or_path: str, **kwargs):
         return download_repo_from_huggingface_hub(
             repo_id=pretrained_model_name_or_path, **kwargs
         )
+
+
+def load_quantized_model_decompressed(pretrained_model_name_or_path: str, **kwargs):
+    """
+    Load compressed-tensors quantized AutoModelForCausalLM.pre_trained(...) model
+     with run_compressed=False.
+
+    Used for Linear modules as Linear, not llmcompressor's custom CompressedLinear
+
+    :param pretrained_model_name_or_path: model name (HF stub) or local path
+
+    """
+    config: CompressedTensorsConfig = AutoConfig.from_pretrained(
+        pretrained_model_name_or_path
+    )
+    quant_method = config.quantization_config.get("quant_method", None)
+
+    # Only add run_compressed for compressed-tensors models
+    if quant_method is not None and quant_method == "compressed-tensors":
+        config.quantization_config["run_compressed"] = False
+
+    return AutoModelForCausalLM.from_pretrained(
+        pretrained_model_name_or_path, config=config, **kwargs
```

Comment:
Wont we get the error you were seeing unless we set the status to Frozen in HFQuant? In which case, the test below will error out?

Reply:
Yes good point. The module set to frozen will be triggered from the HFQuantizer side

Reply:
If this will error until then, we should wait until your change lands in HF and keep this in draft form.

Reply:
yes wait until its merged